app/utilities/schema.py

```diff
@@ -14,7 +15,8 @@
 
 logger = get_logger()
 
-DEFAULT_SCHEMA_DIRS = ["schemas", "test_schemas"]
+SCHEMA_DIR = "schemas"
+DEFAULT_SCHEMA_DIRS = (SCHEMA_DIR, "test_schemas")
```

Comment:
What is the purpose of defining `SCHEMA_DIR` over this?
```suggestion
DEFAULT_SCHEMA_DIRS = ("schemas", "test_schemas")
```

Reply:
So when create app caches the schemas, the `dir` can be passed in as we don't care to cache test schemas.